### DISCUSSION TITLE
How to setup %any remote responder ?

### DISCUSSION THREAD

**Discussion by Alceatraz** [strongswan/strongswan#2419] [Q&A] [answered]
Hello!

I have some question about dynamic networking envioument. As a demo:

As a product, This route can:

- Select local from interface
- Type 0.0.0.0 as remote (same as $any)
- Use ID rather then IP
- Set to Responder mode

Then, This route will auto set local address from pppoe interface and respond ipsec request from source ip ( need no remote config )

![1](https://github.com/user-attachments/assets/885f798b-970f-40b1-95f8-28bd236cdd6a)

---

But when I try to do this with strongswan, Things goes wrong:

First, Let try this on a VPS server with static public IP

```text
conn demo {

	auto = route
	type = tunnel

	left = The public IP
	right = %any

	leftid  = NET_100
	rightid = NET_200

	leftsubnet  = 192.168.100.0/24
	rightsubnet = 192.168.200.0/24
}
```

It failed:

```text
unable to resolve %any, initiate aborted
```

Question:

How to solve this ?

Or, Is there a setting to set strongswan as responder? (I know there is no server/client in Ipsec only peers) Or the router use non-standard implements?

**Comment by strongX509** [comment, marked as answer]
You cannot use **`auto=route`** or **`auto=start`** with a passive responder since the destination address is not known. Only **`auto=add`** is allowed.

**Reply by Alceatraz** [reply]
Thanks, And for someone get in there future:
**`auto = add`** is responder mode.
